### ISSUE TITLE
s/Accessiblity/Accessibility

### ISSUE THREAD

**Issue by beaufortfrancois** [GoogleChrome/lighthouse#1012]
There's a typo in LightHouse options: `Accessiblity` should be `Accessibility`
